src/System.Diagnostics.Debug/tests/System.Diagnostics.Debug.Tests.csproj

```diff
@@ -9,24 +9,23 @@
     <ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>None</ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>
     <NoWarn>1685</NoWarn>
     <NoTargetingPackReferences>true</NoTargetingPackReferences>
-    <WindowsAppContainer>false</WindowsAppContainer>
   </PropertyGroup>
   <!-- Default configurations to help VS understand the configurations -->
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Unix_Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Unix_Release|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Windows_Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Windows_Release|AnyCPU'" />
   <ItemGroup>
-    <Reference Include="System.Diagnostics.Debug" />
-    <Reference Include="System.Runtime" />
-    <Reference Include="System.Runtime.Extensions" />
-    <Reference Include="System.Private.CoreLib" />
-    <Reference Include="System.Threading.Tasks" />
-    <Reference Include="System.Runtime.InteropServices.RuntimeInformation" />
+    <RuntimeReferencedAssemblyNames Include="System.Diagnostics.Debug" />
```

Comment:
I just realized you already had this name in the targeting pack props file but I do still think ReferenceFromRuntime is a better name.